<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camera_aoi_selector</class>
 <widget class="QWidget" name="camera_aoi_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>153</width>
    <height>22</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cmbcamera_aoi"/>
   </item>
   <item>
    <widget class="QComboBox" name="cmbcamera_gain"/>
   </item>
   <item>
    <widget class="QComboBox" name="cmbcamera_bin"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
